packages/binding/src/core/state/EntityState.ts

```diff
@@ -4,6 +4,7 @@ import type { EntityRealmState, EntityRealmStateStub } from './EntityRealmState'
 
 export interface EntityState {
 	entityName: EntityName
+	nullable: boolean | null
```

Comment:
As we discussed yesterday - I think that it's not needed nor makes sense to have it on entity. Nevertheless it is always null since the new argument for `initializeEntityRealm` never passed.

Reply:
I'll give it a quick go, I'll see what comes out of it.

Reply:
@matej21 will do the required changes